schema/artist/index.js

```diff
@@ -88,6 +94,38 @@ const ArtistType = new GraphQLObjectType({
               type: GraphQLInt,
               resolve: ({ auction_lots_count }) => auction_lots_count,
             },
+            forsale_artworks_count: {
+              type: GraphQLInt,
+              resolve: ({ forsale_artworks_count }) => forsale_artworks_count,
+            },
+          },
+        }),
+        resolve: (artist) => artist,
+      },
+      overview: {
+        type: new GraphQLObjectType({
+          name: 'ArtistOverview',
```

Comment:
I'm not so sure this has to exist as a sub-object like this. The bio seems fine on the artist schema itself.